### PR TITLE
docs: fix gramatical and typographical errors

### PR DESCRIPTION
**Pull request by Vanesh37**
## Check List

**Please read and check followings before submitting a PR.**

- [ ] I want to publish my theme on Hexo official website.
  - [ ] I have read the [theme publishing doc](https://hexo.io/docs/themes#Publishing).
  - [ ] `name` is unique.
  - [ ] `link` URL is correct.
  - [ ] `preview` URL is correct.
  - [ ] `preview` URL web site is rendered correctly.
  - [ ] Add a screenshot to `source/themes/screenshots`.
  - [ ] Screenshot filename is same as value of `name`.
  - [ ] Screenshot size is `800 * 500`.
  - [ ] Screenshot file format is `png`.
- [ ] I want to publish my plugin on Hexo official website.
  - [ ] I have read the [plugin publishing doc](https://hexo.io/docs/plugins#Publishing).
  - [ ] `name` is unique.
  - [ ] `link` URL is correct.
- [x] Others (Update, fix, translation, etc...)
  - Languages:
  - [x] `en` English
  - [ ] `ru` Russian
  - [ ] `zh-cn` simplified Chinese
  - [ ] `zh-tw` traditional Chinese

> **Note**
>
> No need to translate `ko`, `pt-br`, `th` languages if you submit a PR for documentation.

> **Warning**
>
> `ko`, `pt-br`, `th` languages document are inactive. Please, submit issue if you want to continuous contribution to them.

Minor improvements in files under docs/ directory

### CODE REVIEW

File: source/docs/contributing.md [hunk 1]
```diff
@@ -90,14 +90,14 @@ $ hexo server
...
-4. Copy `en.yml` in `themes/navy/languages` and rename to the language name (all lower case).
+4. Copy `en.yml` in `themes/navy/languages` and rename it to the language name (all lower case).
 
 ## Reporting Issues
 
 When you encounter some problems when using Hexo, you can find the solutions in [Troubleshooting](troubleshooting.html) or ask me on [GitHub](https://github.com/hexojs/hexo/issues) or [Google Group](https://groups.google.com/group/hexo). If you can't find the answer, please report it on GitHub.
 
 1. Represent the problem in [debug mode](commands.html#Debug_mode).
-2. Follow the steps from issue template to provide debug message and version when submitting a new issue at GitHub.
+2. Follow the steps from issue the template to provide a debug message and version when submitting a new issue at GitHub.
```
Comment (uiolee) [inline source/docs/contributing.md:100]:
```suggestion
2. Follow the steps from the issue template to provide a debug message and version when submitting a new issue at GitHub.
```

File: source/docs/configuration.md [hunk 2]
```diff
@@ -122,11 +122,11 @@ Setting | Description | Default
 {% note info updated_option %}
 `updated_option` controls the `updated` value when not provided in the front-matter:
 
-- `mtime`: Use file modification date as `updated`. It is the default behavior of Hexo since 3.0.0
+- `mtime`: Use file modification date as `updated`. It has been the default behaviour of Hexo since 3.0.0
```
Comment (uiolee) [inline source/docs/configuration.md:125]: `behavior` not bad

File: source/docs/syntax-highlight.md [hunk 3]
```diff
@@ -22,7 +22,7 @@ code snippet
 ```
 ````
 
-The third syntax is a Markdown's fenced code block syntax and Hexo extends it to support more features. Check out [Tag Plugin Docs](tag-plugins#Code-Block) to find out options available.
+The third syntax is Markdown's fenced code block syntax and Hexo extends it to support more features. Check out [Tag Plugin Docs](tag-plugins#Code-Block) to find out the options available.
```
Comment (uiolee) [inline source/docs/syntax-highlight.md:25]:
```suggestion
The third syntax is Markdown's fenced code block syntax, and Hexo extends it to support more features. Check out [Tag Plugin Docs](tag-plugins#Code-Block) to find out the options available.
```

File: source/docs/tag-plugins.md [hunk 4]
```diff
@@ -3,7 +3,7 @@ title: Tag Plugins
 ---
 Tag plugins are different from post tags. They are ported from Octopress and provide a useful way for you to quickly add specific content to your posts.
 
-Although you can write your posts in any formats, but the tag plugins will always be available and syntax remains the same.
+Although you can write your posts in any format, but the tag plugins will always be available and syntax remains the same.
```
Comment (uiolee) [inline source/docs/tag-plugins.md:6]:
```suggestion
Although you can write your posts in any format, the tag plugins will always be available and syntax remains the same.
```

File: source/docs/troubleshooting.md [hunk 5]
```diff
@@ -98,7 +98,7 @@ Make sure you have [set up git](https://help.github.com/articles/set-up-git) on
 
 ### Error: ENOENT: no such file or directory
 
-If you get an error like `Error: ENOENT: no such file or directory` it's probably due to to mixing uppercase and lowercase letters in your tags, categories, or filenames. Git cannot automatically merge this change so it breaks the automatic branching.
+If you get an error like `Error: ENOENT: no such file or directory` it's probably due to mixing uppercase and lowercase letters in your tags, categories, or filenames. Git cannot automatically merge this change so it breaks the automatic branching.
```
Comment (uiolee) [inline source/docs/troubleshooting.md:101]:
```suggestion
If you get an error like `Error: ENOENT: no such file or directory` it's probably due to mixing uppercase and lowercase letters in your tags, categories, or filenames. Git cannot automatically merge this change, so it breaks the automatic branching.
```

File: source/docs/troubleshooting.md [hunk 6]
```diff
@@ -178,7 +178,7 @@ When you can't get any command except `help`, `init` and `version` to work and y
 
 ## Escape Contents
 
-Hexo uses [Nunjucks] to render posts ([Swig] was used in older version, which share a similar syntax). Content wrapped with `{{ }}` or `{% %}` will get parsed and may cause problems. You can skip the parsing by wrapping it with the [`raw`](/docs/tag-plugins#Raw) tag plugin, single backtick ```` `{{ }}` ```` or triple backtick.
+Hexo uses [Nunjucks] to render posts ([Swig] was used in older version, which shares a similar syntax). Content wrapped with `{{ }}` or `{% %}` will get parsed and may cause problems. You can skip the parsing by wrapping it with the [`raw`](/docs/tag-plugins#Raw) tag plugin, single backtick ```` `{{ }}` ```` or triple backtick.
```
Comment (uiolee) [inline source/docs/troubleshooting.md:181]:
```suggestion
Hexo uses [Nunjucks] to render posts ([Swig] was used in the older version, which shares a similar syntax). Content wrapped with `{{ }}` or `{% %}` will get parsed and may cause problems. You can skip the parsing by wrapping it with the [`raw`](/docs/tag-plugins#Raw) tag plugin, a single backtick ```` `{{ }}` ```` or a triple backtick.
```

File: source/docs/writing.md [hunk 7]
```diff
@@ -67,8 +67,8 @@ Placeholder | Description
 
 ## Supported Formats
 
-Hexo support posts written in any format, as long as the corresponding renderer plugin is installed.
+Hexo supports posts written in any format, as long as the corresponding renderer plugin is installed.
 
 For example, Hexo has `hexo-renderer-marked` and `hexo-renderer-ejs` installed by default, so you can write your posts in `markdown` or in `ejs`. If you have `hexo-renderer-pug` installed, then you can even write your post in pug template language.
 
-You can rename your posts and change to file extension from `.md` to `.ejs`, then Hexo will use `hexo-renderer-ejs` to render that file, so do the other formats.
+You can rename your posts and change to file extension from `.md` to `.ejs`, then Hexo will use `hexo-renderer-ejs` to render that file, and so do the other formats.
```
Comment (uiolee) [inline source/docs/writing.md:74]:
```suggestion
You can rename your posts and change the file extension from `.md` to `.ejs`, then Hexo will use `hexo-renderer-ejs` to render that file, and so do the other formats.
```

**Review by uiolee — COMMENTED** [review]
thank you for your patient work

**Comment by Vanesh37** [comment]
hi @uiolee  , the changes are done according to your suggestions please check and merge it

**Review by uiolee — APPROVED** [review]

**Pull request merged**